### PR TITLE
fix missing files for boostrap incorporation filing

### PR DESCRIPTION
**Pull request by BrandonSharratt**
*Issue:*https://github.com/bcgov/entity/issues/24541

*Description of changes:*
The bootstrap filing loader was not getting certain things from the correct spot for instance, the documentsLink so I fixed them as an || incase there was some case I was missing where they are in that field.

Additionally there was no space for future effective filings between the filing name and PAID so I added one.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-89-x16xbutk.web.app

File: src/components/bcros/filing/common/futureEffective/Paid.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <span v-else-if="isTypeDissolutionVoluntary">{{ $t('text.filing.futureEffectiveDissolution') }}</span>
     <span v-else>{{ $t('text.filing.futureEffectiveFiling') }}</span>
     <span>
-      {{ $t('text.filing.paid') }}
+      {{ ' ' + $t('text.filing.paid') }}
```
Comment (severinbeauvais) [inline src/components/bcros/filing/common/futureEffective/Paid.vue:9]: FYI, you can also use `&nbsp;` to add a space.

**Review by severinbeauvais — APPROVED** [review]
Thanks for the sample business. Looks good!

**Pull request merged**